### DISCUSSION TITLE
Patching Gaps in the CIS Windows 11 Benchmark - Level 2 Windows 11 - odds+endpoints

### DISCUSSION THREAD

**Discussion by giscus** [ennnbeee/oddsandendpoints-web#6] [Comments]
# Patching Gaps in the CIS Windows 11 Benchmark - Level 2 Windows 11

Updating the Center for Internet Security (CIS) benchmark build kit for Windows 11 in Microsoft Intune for Level 2 settings.

https://memv.ennbee.uk/posts/windows-cis-patching-gaps-part4/

**Comment by alevalesantos** [comment]
Hi Nick

I used your settings with my configurations and I created a CIS Template using a json file to import in a only one settings.
If you would like to check, please go the discussion in https://github.com/ennnbeee/ennnbeee.github.io/discussions/5#discussioncomment-10222638
